controller/workitem.go

```diff
@@ -55,86 +53,6 @@ func NewWorkitemController(service *goa.Service, db application.DB, config WorkI
 		config:     config}
 }
 
-// List runs the list action.
```

Comment:
some operations are now bound to `/api/workitems/` while others are still bound to `/api/spaces/<space_id>/workitem/`. When re-generating the controllers code(after the changes in the design), Goa created a new controller and controller file. Hence, we now have 2 controllers: 
-
 `WorkItemController` (`WorkItem`, singular): for operations on a single work item (show, update and delete)
- `WorkItemsController` (`WorkItems`, plural): for operations on multiple work item (create, list, reorder)